### DISCUSSION TITLE
Windows 10 behavior with 100% CPU for many hours?

### DISCUSSION THREAD

**Discussion by jbeale1** [ultralytics/yolov5#9729] [General]
Maybe unrelated to Yolo5, but just in case... I tried running the Yolo5 training example described at
https://blog.paperspace.com/train-yolov5-custom-data/
using CPU on Windows 10 laptop (Dell Precision 5550, CPU is i7-10750H with 6 cores). Running 100 epochs worked as expected, mAP_0.5 reached above 0.95 and the CPU-only training took about 12 hours.  
After this run, and much later (the following day) when running other, unrelated programs, I found my laptop would run high-CPU programs slower.  Programs that normally used 100% CPU before, now only use about 20% CPU and run much more slowly. However after rebooting the computer, it was back to normal, and the programs ran at their usual speed and reached 100% CPU again.  Is this an expected Windows thing that it will throttle how many CPU cores are active semi-permanently, until machine restarts?   The laptop had been idle overnight and was physically cool, and was plugged in to wall power throughout.

**Comment by glenn-jocher** [comment]
@jbeale1 I'm not aware of anything of this nature.
